README.md

```diff
@@ -1 +1,16 @@
-# 3pa-register-api
\ No newline at end of file
+# 3pa-register-api
+The Third Party Agent Register application is an internal service.
+This service is responsible for collecting third party agent details and the issuance of an agent assurance code.
+
+The service integrates with a number of internal systems. 
+
+Requirements
+------------
+* Java 11
+* [Git](https://git-scm.com/downloads)
+* [Maven](https://maven.apache.org/download.cgi)
+* [3pa-register-api](https://github.com/companieshouse/3pa-register-api)
```

Comment:
this ought to be the 3pa-web yeah?  Though I suppose the api could stand alone..